lib/ssh/src/ssh.erl

```diff
@@ -549,8 +549,8 @@ stop_listener(Address, Port, Profile) ->
 %% Description: Stops the listener and all connections started by
 %% the listener.
 %%--------------------------------------------------------------------
--spec stop_daemon(DaemonRef::daemon_ref()) -> ok.
-
+-spec stop_daemon(DaemonRef::daemon_ref()) ->
+          supervisor:terminatechild_ret() | undefined.
```

Comment:
Do we really want the function to return undefined?  I always though of this function as similar to  lists:delete(),  but without a new value, that is if you delete something that does not exist then it is ok and if it exists it will be deleted.

Reply:
If we do want to care about informing the user of giving suppling a pid as input that is not an ssh_deamon we should not use suprvisor return value
as it has return values that will not be possible as we know what type of supervisor that we got.

Reply:
I think I would prefer to leave it as is for now.